### PR TITLE
ARROW-194 Convert top level docs to Markdown

### PR DESCRIPTION
**Pull request by blink1073**

### CODE REVIEW

File: bindings/python/README.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+- Documentation: Available at
+[mongo-arrow.readthedocs.io](https://mongo-arrow.readthedocs.io/en/latest/).
+
+- Author: Prashant Mital
+
+**PyMongoArrow** is a companion library to PyMongo that contains tools
+for loading MongoDB query result sets as Apache Arrow tables, Pandas
+DataFrames or NumPy arrays.
+
+``` pycon
```
Comment (ShaneHarvey) [inline bindings/python/README.md:17]: extra space between \` and pycon here

File: bindings/python/README.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
+# PyMongoArrow
+
+- Info: A companion library to PyMongo that makes it easy to move data between
```
Comment (ShaneHarvey) [inline bindings/python/README.md:3]: Is there a nicer way to render this info rather than a bulleted list? The rst version was in a table.

**Review by ShaneHarvey — CHANGES_REQUESTED** [review]

File: bindings/python/README.md (hunk 2 above)
Comment (blink1073) [reply bindings/python/README.md:3]: GFM does not support tables with no headers.

File: bindings/python/README.md (hunk 1 above)
Comment (blink1073) [reply bindings/python/README.md]: fixed

File: bindings/python/README.md (hunk 2 above)
Comment (ShaneHarvey) [reply bindings/python/README.md:3]: How about we just remove this section?

Comment (blink1073) [reply bindings/python/README.md]: Done, I've opened https://jira.mongodb.org/browse/ARROW-196 to track adding badges instead.

**Review by ShaneHarvey — APPROVED** [review]

**Pull request merged**